package.json

```diff
@@ -88,7 +88,7 @@
     "multiselect": "~0.9.12",
     "number_helpers": "^0.1.1",
     "patternfly": "^3.58.0",
-    "patternfly-react": "^2.29.0",
+    "patternfly-react": "^2.30.6",
```

Comment:
https://github.com/theforeman/foreman-packaging/pull/3540 now updates to the latest (2.31.0) because that's allowed by `^2.30.6`